EIPS/eip-future_salaries.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: <to be assigned>
+title: Giving Ethereum fees to Future Salaries
+author: Victor Porton <porton@narod.ru>
+status: Draft
```

Comment:
```suggestion
discussions-to: <URL>
status: Draft
```
Create a thread over at https://ethereum-magicians.org/ for discussing this proposal and link it here, or if that isn't possible or desirable for some reason you can create a GitHub issue (not a PR) for discussion and link it here.  (Ethereum Magicians is strongly preferred).